### PR TITLE
Fix missing thumbnails for notebook gallery

### PR DESCRIPTION
**Pull request by fziegner**
This PR addresses #426. The thumbnails are now saved in the _static directory instead of being selected from the notebooks. This also makes it possible to choose specific thumbnails for the gallery. Some of the thumbnails are still suboptimal so I'll be looking for a way to improve them.

* [ ] Have you followed our guidelines in CONTRIBUTING.md? 
* [ ] Have you self-reviewed your code and corrected any misspellings? 
* [ ] Have you written documentation that is easy to understand?
* [ ] Have you written descriptive commit messages? 
* [ ] Have you added NumPy docstrings for newly added functions? 
* [ ] Have you formatted your code using black? 
* [ ] If you have introduced a new functionality, have you added adequate unit tests?
* [ ] Have all tests passed in your local clone? 
* [ ] If you have introduced a new functionality, have you added an example notebook?
* [ ] Have you kept your pull request small and limited so that it is easy to review? 
* [ ] Have the newest changes from this branch been merged?

### CODE REVIEW

**Comment by fsenf** [comment]
https://tobac--428.org.readthedocs.build/en/428/

**Comment by fsenf** [comment]
I included a colorful set of thumbnail images. Hope that looks nice now ...

**Comment by fsenf** [comment]
@w-k-jones : this documentation piece is ready. Would you like to act as 2nd reviewer?

PS: Unfortunately, the website under the link above did not update to show the newest image versions....

**Review by w-k-jones — APPROVED** [review]
Looks great! The new images are showing up for me in the test build. Happy for this to be merged

edit: just double checked, are the images for the segmentation basics and linking basics swapped?

File: doc/conf.py [hunk 1]
```diff
@@ -79,3 +79,18 @@ def setup(app):
...
 napoleon_type_aliases = None
 napoleon_attr_annotations = True
+
+nbsphinx_thumbnails = {
+    "examples/Basics/Idealized-Case-1_Tracking-of-a-Test-Blob-in-2D": "_static/thumbnails/Basics_Idealized-Case-1_Tracking-of-a-Test-Blob-in-2D_Thumbnail.png",
+    "examples/Basics/Idealized-Case-2_Two_crossing_Blobs": "_static/thumbnails/Basics_Idealized-Case-2_Two_crossing_Blobs_Thumbnail.png",
+    "examples/Basics/Methods-and-Parameters-for-Feature-Detection_Part_1": "_static/thumbnails/Basics_Methods-and-Parameters-for-Feature-Detection_Part_1_Thumbnail.png",
+    "examples/Basics/Methods-and-Parameters-for-Feature-Detection_Part_2": "_static/thumbnails/Basics_Methods-and-Parameters-for-Feature-Detection_Part_2_Thumbnail.png",
+    "examples/Basics/Methods-and-Parameters-for-Segmentation": "_static/thumbnails/Basics_Methods-and-Parameters-for-Linking_Thumbnail.png",
+    "examples/Basics/Methods-and-Parameters-for-Linking": "_static/thumbnails/Basics_Methods-and-Parameters-for-Segmentation_Thumbnail.png",
```
Comment (w-k-jones) [inline doc/conf.py:89]:
```suggestion
    "examples/Basics/Methods-and-Parameters-for-Segmentation": "_static/thumbnails/Basics_Methods-and-Parameters-for-Segmentation_Thumbnail.png",
    "examples/Basics/Methods-and-Parameters-for-Linking": "_static/thumbnails/Basics_Methods-and-Parameters-for-Linking_Thumbnail.png",
```

**Review by w-k-jones — CHANGES_REQUESTED** [review]
One quick change then I'm happy to merge

File: doc/conf.py (hunk 1 above)
Comment (fsenf) [reply doc/conf.py]: Very well spotted! Great job!

Comment (fsenf) [reply doc/conf.py]: Hi Will, I was too fast and did the correction already locally in the back. Could you withdraw your suggestions or what should I do to resolve it?

**Review by w-k-jones — APPROVED** [review]

**Pull request merged**